test/libsolidity/syntaxTests/array/length/not_too_large.sol

```diff
@@ -7,3 +7,4 @@ contract C {
     }
 }
 // ----
+// TypeError: (226-234): Type too large for memory.
```

Comment:
The name of the test is "not_too_large"

Reply:
I'll move it to the iceRegressionTests and rename it.

Reply:
But is there a test that checks the same thing?